app/assets/stylesheets/vendor/_select2.sass

```diff
@@ -0,0 +1,422 @@
+.select2-container
```

Comment:
This one should go to vendor/assets/stylesheets/select2.sass. I put it in wrong place. Additionally app/assets/stylesheets/select2.css needs to be removed if we have .sass version.